docs/config.md

```diff
@@ -201,6 +201,14 @@ version = 2
     [plugins."io.containerd.grpc.v1.cri".registry.mirrors]
       [plugins."io.containerd.grpc.v1.cri".registry.mirrors."docker.io"]
         endpoint = ["https://registry-1.docker.io", ]
+
+  # 'plugins."io.containerd.grpc.v1.cri".image_decryption' contains config related
+  # to the handling decryption of encrypted container images.
+  [plugins."io.containerd.grpc.v1.cri".image_decryption]
+    # key_model sets how the trust model of where keys should reside for the
+    # decryption of encrypted container images. The default is "node".
+    # The docs/encryption.md file provides further information of key models.
```

Comment:
it provides an example but not a formal definition for the config where each field is listed and described... Let's provide a link to that doc for an overall description but put the formal detail for each field here.

Reply:
Let me see if I understand this.. something like this?

```
    # key_model defines the name of the key model used for how the cri obtains
    # keys used for decryption of encrypted container images.
    # Set of available string options: {"node"}
    # The docs/encryption.md file provides further information of key models.
    key_model = "node"
```

Reply:
```()[]``` link

Reply:
here's the doc for it... 
https://github.com/containerd/containerd/blob/master/docs/stream_processors.md

Reply:
perhaps something like... ```# see (stream processors)[link] for definition, and (encryption stream config example)[] for the required stream processors to provide decryption function?

Reply:
thoughts?

Reply:
Yea, i like that. I think that's the cleanest.